### DISCUSSION TITLE
Next updated ISO | (Illegal snapshot issues)

### DISCUSSION THREAD

**Discussion by Nicodile** [SpiralLinux/SpiralLinux-project#487] [Q&A] [answered]
Hello All,

Thank you for creating a wonderful distro Gecko firstly.

Anyway, I recently wanted my learning laptop to be very stable, so I no longer have to deal with updating things and breaking. However, when I downloaded Spiral it started giving me an error (I believe this is a few days after I dealt with the error) but after my first update it kept saying illegal snapshot. I tried purging all snapshots and used a few openSUSE guides, but nothing worked, so I moved on towards MX Linux for now to test and see if I liked Debian in general. My question is really when will the next updated ISO be out, or is there any cadence for that.

TLDR: I updated and got an "illegal Snapshot" error, moved on to MX Linux for now after small troubleshooting. When will new updated ISO be out.

Thanks again.

Edit:
Figured out the issue, I was able to fix this by upgrading Nala linux. For some reason the bookworm was having issue and this was fixed by installing the volian scar repo and the updated version worked better. I additionally deleted all snapshots.

**Comment by geckolinux** [comment]
Hi there, I would need a lot more details about your installation layout and what you tried to do with the snapshot system before the error started appearing, and where and when the error appeared.

There is no need yet for an updated set of SpiralLinux ISOs, as there are no major changes in Debian 12 since the last SpiralLinux release, and no major bugs in the current SpiralLinux ISOs.

**Reply by Nicodile** [reply]
Thank you for the quick reply, I posted some screenshots. It seems the error happens after upgrading to the most recent update and then doing a full-upgrade to clean out the unused portion (kernel for this example). Then installing two programs (variety and qbittorent in this example causes illegal snapshot)  am using nala to install, However this error also persists with apt install as well. 

![Screenshot_20240927_080147](https://github.com/user-attachments/assets/6beb3e5e-ae78-4ef4-97af-922d0cc25965)


![Screenshot_20240927_080443](https://github.com/user-attachments/assets/e8fbfb72-0081-4b5c-acf8-86e131be4fdb)

**Reply by geckolinux** [reply, marked as answer]
Hi again, are you maybe running on a small virtual machine disk that is out of space? If not then I would assume that `nala` is the problem, you could try deleting all the snapshots and then running some package operations with `apt` or Synaptic.

**Comment by Nicodile** [comment]
its on a 500GB SSD and on real hardware. But it is strange, I deleted the snapshots again as I was testing it before I posted and now it is working again. Always works once you complain. 

I appreciate you taking the time out of your day.

**Reply by geckolinux** [reply]
Glad it's working now, thanks for confirming that.
